groclient/cli.py

```diff
@@ -1,13 +1,51 @@
+from random import random
 import argparse
 import getpass
 import os
 import sys
+import unicodecsv
 
 import groclient.lib
 import groclient.cfg
 from groclient import GroClient
 
-OUTPUT_FILENAME = "gro_client_output.csv"
+
+def pick_random_data_series(client):  # pragma: no cover
```

Comment:
Could you please explain a bit the use case of picking random data series ? 
Even if random() cuts down the number, we are still returning a lot of series.

Reply:
The `gro_client` command line tool is for the most part a toy example, so the randomize thing is mostly just for fun. If you don't specify any metrics or items etc. and run `gro_client` then it just picks something random. So you can end up surprised at what obscure series exist in Gro.

> Even if random() cuts down the number, we are still returning a lot of series.

It only picks a single series. It prints out the data points for that series to the console, so there may be many points, but they all belong to the same series.

Reply:
For example:

```sh
$ gro_client
Data series have some historical regions, see https://developers.gro-intelligence.com/faq.html
annual Export Volume (mass) of Fruit, prepared, other in Netherlands:
1992-01-01T00:00:00.000Z-1992-12-31T00:00:00.000Z: 20789 kg
1997-01-01T00:00:00.000Z-1997-12-31T00:00:00.000Z: 13000 kg
1998-01-01T00:00:00.000Z-1998-12-31T00:00:00.000Z: 500 kg
1999-01-01T00:00:00.000Z-1999-12-31T00:00:00.000Z: 20390 kg
2000-01-01T00:00:00.000Z-2000-12-31T00:00:00.000Z: 21310 kg
2001-01-01T00:00:00.000Z-2001-12-31T00:00:00.000Z: 9614 kg
2002-01-01T00:00:00.000Z-2002-12-31T00:00:00.000Z: 12472 kg
2003-01-01T00:00:00.000Z-2003-12-31T00:00:00.000Z: 65478 kg
2004-01-01T00:00:00.000Z-2004-12-31T00:00:00.000Z: 535086 kg
2005-01-01T00:00:00.000Z-2005-12-31T00:00:00.000Z: 222636 kg
2006-01-01T00:00:00.000Z-2006-12-31T00:00:00.000Z: 227139 kg
2007-01-01T00:00:00.000Z-2007-12-31T00:00:00.000Z: 255605 kg
2008-01-01T00:00:00.000Z-2008-12-31T00:00:00.000Z: 308792 kg
2009-01-01T00:00:00.000Z-2009-12-31T00:00:00.000Z: 412575 kg
2010-01-01T00:00:00.000Z-2010-12-31T00:00:00.000Z: 288542 kg
2011-01-01T00:00:00.000Z-2011-12-31T00:00:00.000Z: 372603 kg
2012-01-01T00:00:00.000Z-2012-12-31T00:00:00.000Z: 291402 kg
2013-01-01T00:00:00.000Z-2013-12-31T00:00:00.000Z: 667700 kg
2014-01-01T00:00:00.000Z-2014-12-31T00:00:00.000Z: 377676 kg
2015-01-01T00:00:00.000Z-2015-12-31T00:00:00.000Z: 569061 kg
2016-01-01T00:00:00.000Z-2016-12-31T00:00:00.000Z: 445805 kg
2017-01-01T00:00:00.000Z-2017-12-31T00:00:00.000Z: 381603 kg
2018-01-01T00:00:00.000Z-2018-12-31T00:00:00.000Z: 601239 kg
2019-01-01T00:00:00.000Z-2019-12-31T00:00:00.000Z: 1120789 kg
```

You can see the series it picked is "annual Export Volume (mass) of Fruit, prepared, other in Netherlands" and those are all the data points for that series

Reply:
Got it, thanks